pinc/base.inc

```diff
@@ -1,84 +1,25 @@
 <?php
+// This file sets up code infrastructure for pages that output HTML.
```

Comment:
s/HTML/content/, since presumably this will include raw JSON too.

Reply:
No, `base.inc` is just for HTML files.